### PR TITLE
Replace ILDCellID0 with LCTrackerCellID

### PR DESCRIPTION
**Pull request by oviazlo**
to address issue https://github.com/iLCSoft/LCIO/issues/4

### CODE REVIEW

File: Tracking/src/ClicEfficiencyCalculator.cc [hunk 1]
```diff
@@ -20,7 +20,7 @@
 #include <IMPL/TrackImpl.h>
 
 #include <UTIL/CellIDEncoder.h>
-#include <UTIL/ILDConf.h>
+#include "UTIL/LCTrackerConf.h"
```
Comment (andresailer) [inline Tracking/src/ClicEfficiencyCalculator.cc]: Please keep the angle brackets

File: Tracking/src/HitResiduals.cc [hunk 2]
```diff
@@ -11,7 +11,7 @@
 #include "MarlinTrk/IMarlinTrack.h"
 
 #include <UTIL/BitField64.h>
-#include <UTIL/ILDConf.h>
+#include "UTIL/LCTrackerConf.h"
```
Comment (andresailer) [inline Tracking/src/HitResiduals.cc]: Please keep the angle brackets

File: Tracking/src/TrackChecker.cc [hunk 3]
```diff
@@ -23,7 +23,7 @@
 #include <IMPL/TrackImpl.h>
 
 #include <UTIL/CellIDEncoder.h>
-#include <UTIL/ILDConf.h>
+#include "UTIL/LCTrackerConf.h"
```
Comment (andresailer) [inline Tracking/src/TrackChecker.cc]: Please keep the angle brackets

**Comment by oviazlo** [comment]
fixed

**Comment by andresailer** [comment]
we get
```
 ***********************************************
 A runtime error occured - (uncaught exception):
      The encoding string was already accessed! Changing it now will lead to inconsistencies! Fix your code!
 Marlin will have to be terminated, sorry.
 ***********************************************
```
during reconstruction, needs to be investigated, where we access the string already.\
Probably conformal tracking
https://github.com/iLCSoft/ConformalTracking/pull/1/files#diff-8fa0895da17afe71e3e8dea58f6c83ccR114

**Comment by andresailer** [comment]
with ilcsoft/lcio#19, ilcsoft/clupatra#7 ilcsoft/marlinreco#4 this should be running again.

**Pull request merged**